app/models/pensions/selfEmployment/Business.scala

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
selfEmployment needs to be a separate package in models. It is not part of pensions